chat-plugins/info.js

```diff
@@ -1065,7 +1065,13 @@ var commands = exports.commands = {
 	weakness: function (target, room, user) {
 		if (!target) return this.parse('/help weakness');
 		if (!this.canBroadcast()) return;
-		var targets = target.split(/[ ,\/]/);
```

Comment:
Ok, that works
<code>target.trim().split(/\s*[,\/ ]\s*/)</code>

All these regexes look like very funny unicode faces